setup.py

```diff
@@ -32,7 +32,7 @@ def getversion():
     long_description=read('README.rst'),
     packages=find_packages(exclude=('test*',)),
     python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
-    install_requires=['pytest>=3.5.0', 'enum34==1.1.6; python_version <= "3.4"', 'attrs'],
```

Comment:
actually, what if we did instead `enum34;python_version=="2.7"` (since I don't think we care about python3.3)

also we shouldn't pin (`==`) enum34 since we're a library (to maximize compatibility)

Reply:
The issue is setuptools no longer supports 3.4 and they only recently added the `;python_version` field, so even if you update setuptools, it still breaks when installing this.

Agree on unpinning enum34 tho

Reply:
`==` has been supported much longer than `<` or `<=`

Reply:
and if you want maxmium compatibility, you can use this syntax (supported even further back!)

https://github.com/pre-commit/pre-commit/blob/a5f312e4e10337cc0306370c28995ab75478ad89/setup.py#L49-L51